ConcernsCaseWork/ConcernsCaseWork.Tests/Security/RbacManagerTests.cs

```diff
@@ -25,12 +25,29 @@ public void WhenGetDefaultUsers_Return_Users()
 			var rbacManager = new RbacManager(BuildConfiguration(), mockUserRoleCachedService.Object, mockLogger.Object);
 			
 			// act
-			var defaultUsers = rbacManager.GetDefaultUsers();
+			var defaultUsers = await rbacManager.GetSystemUsers();
 
 			// assert
 			Assert.That(defaultUsers, Is.Not.Null);
 		}
-		
+
+		[Test]
+		public async Task WhenGetSystemUsers_Return_Users_Except_For_Exclusions()
+		{
+			// arrange
+			var mockUserRoleCachedService = new Mock<IUserRoleCachedService>();
+			var mockLogger = new Mock<ILogger<RbacManager>>();
+
+			var rbacManager = new RbacManager(BuildConfiguration(), mockUserRoleCachedService.Object, mockLogger.Object);
+
+			// act
+			var defaultUsers = await rbacManager.GetSystemUsers("user3");
+
+			// assert
+			Assert.That(defaultUsers, Is.Not.Null);
+			Assert.IsFalse(defaultUsers.Contains("user3"));
```

Comment:
This test isn't checking that users are returned (just checking the exclusions).

Reply:
updated test